src/gtclang/Frontend/GTClangASTConsumer.cpp

```diff
@@ -276,7 +276,14 @@ void GTClangASTConsumer::HandleTranslationUnit(clang::ASTContext& ASTContext) {
   // Write file to disk
   DAWN_LOG(INFO) << "Writing file to disk... ";
   std::error_code ec;
+#if  __clang_major__ == 6
```

Comment:
this is not working since we are compiling with gcc, we have to do this via cmake